src/include/migraphx/op/contiguous.hpp

```diff
@@ -48,7 +48,7 @@ struct contiguous
     {
         check_shapes{inputs, *this, true}.has(1);
         auto s0 = inputs.front();
-        if(s0.dynamic() or s0.standard())
+        if(s0.dynamic() or s0.scalar())
```

Comment:
Contiguous after scalar should be a standard shape nor a scalar shape.

Reply:
ok reverted. However this test case in op_shape_test.cpp suggests that contiguous after scalar is scalar:

TEST_CASE(contiguous_shape_scalar)
{
    migraphx::shape output{migraphx::shape::float_type};
    migraphx::shape input{migraphx::shape::float_type};
    expect_shape(output, migraphx::make_op("contiguous"), input);
}